pkg/registry/resource/resourceclaim/strategy.go

```diff
@@ -149,7 +149,8 @@ func (resourceclaimStatusStrategy) PrepareForUpdate(ctx context.Context, obj, ol
 func (resourceclaimStatusStrategy) ValidateUpdate(ctx context.Context, obj, old runtime.Object) field.ErrorList {
 	newClaim := obj.(*resource.ResourceClaim)
 	oldClaim := old.(*resource.ResourceClaim)
-	return validation.ValidateResourceClaimStatusUpdate(newClaim, oldClaim)
+	opts := validation.ValidationOptionsForResourceClaimStatus(newClaim, oldClaim)
```

Comment:
@pohly @thockin this is a brand new API we're just promoting to beta in this release... did we know we were promoting to beta with a validation gap, and can we just close this unconditionally in beta?